### DISCUSSION TITLE
Alerts in Brand Colors

### DISCUSSION THREAD

**Discussion by flomero** [saadeghi/daisyui#2364] [Ideas / feature requests]
I would love to have Alerts available in brand colours (primary, secondary, neutral, ...). Of course, you can achieve the same effect using `bg-primary text-primary-content border-primary/20`, but it would be easier and more intuitive if `class="alert alert-primary"` would just work. In most use cases alerts should reflect what they are used for, but I ran into a few cases where a neutral alert/toast was needed (e.g. client wants their primary colour to be used). I would like to contribute a corresponding pull request if this proposal is accepted 🙌🏼

**Comment by saadeghi** [comment]
> I ran into a few cases where a neutral alert/toast was needed

By default, the `alert` class gives you a `base-200` color which is not promoting any specific state (`info`, `success`, `warning`, `error`)  


> e.g. client wants their primary colour to be used

That's not a good idea to use primary color for an alert. So I don't think having that would be a good decision to add those variants for alert. However as you know, you can always customize the components using utility classes or custom CSS.
